feel/feelcore/environment.cpp

```diff
@@ -2211,6 +2211,7 @@ Environment::expand( std::string const& expr )
     boost::replace_all( res, "${datadir}", dataDir );
     boost::replace_all( res, "${exprdbdir}", exprdbDir );
     boost::replace_all( res, "${h}", std::to_string(doption("gmsh.hsize") ) );
+    boost::replace_all( res, "${testsuite_srcdir}", topSrcDir + "/testsuite/" );
 
```

Comment:
This is not a good way because topSrcDir doesnt work in a install process and also testsuite is a subproject which can be compile out of feel++ lib.
Note : some keywords like topSrcDir will be remove of this function.